### PR TITLE
Connect controllers to user profile page/Connect controllers to edit …

### PR DESCRIPTION
**Pull request by GetFire**
…user profile page

### CODE REVIEW

File: src/main/java/net/greatstart/controllers/UserController.java [hunk 1]
```diff
@@ -58,14 +64,26 @@ public ModelAndView editProfile(@PathVariable long id, Principal principal) {
...
         if (errors.hasErrors()) {
             return new ModelAndView(EDIT_PROFILE);
         }
+        if (!file.isEmpty()) {
+            byte[] content = null;
+            try {
+                content = file.getBytes();
+                dtoUser.setPhoto(content);
+            } catch (IOException e) {
+                e.printStackTrace();
```
Comment (Sheptytskyid) [inline src/main/java/net/greatstart/controllers/UserController.java]: MINOR Use a logger to log this exception. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1148)

File: src/main/java/net/greatstart/dto/DtoUserProfile.java [hunk 2]
```diff
@@ -14,4 +14,6 @@
     private String address;
     private String phoneNumber;
     private String email;
+    private String lastName;
```
Comment (Sheptytskyid) [inline src/main/java/net/greatstart/dto/DtoUserProfile.java:17]: MAJOR Remove this unused "lastName" private field. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)

File: src/main/java/net/greatstart/dto/DtoUserProfile.java [hunk 3]
```diff
@@ -14,4 +14,6 @@
...
+    private byte[] photo;
```
Comment (Sheptytskyid) [inline src/main/java/net/greatstart/dto/DtoUserProfile.java:18]: MAJOR Remove this unused "photo" private field. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by zozich — APPROVED** [review]
Fix logger

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**